tests/api.c

```diff
@@ -18574,6 +18574,76 @@ static int test_wc_AesGcmEncryptDecrypt(void)
 
 } /* END test_wc_AesGcmEncryptDecrypt */
 
+/*
+ * test function for mixed (one-shot encrpytion + stream decryption) AES GCM
+ * using a long IV (older FIPS does NOT support long IVs).  Relates to zd15423
+ */
+static int test_wc_AesGcmMixedEncDecLongIV(void)
+{
+    int ret = TEST_SKIPPED;
+
+#if  (!defined(HAVE_FIPS) || \
+      (defined(HAVE_FIPS_VERSION) && (HAVE_FIPS_VERSION >= 2))) && \
+     !defined(NO_AES) && defined(HAVE_AESGCM) && defined(WOLFSSL_AESGCM_STREAM)
+
+    Aes     aesEnc, aesDec;
+    byte    key[] =
```

Comment:
Formatting -> `byte key[] = {`. Make these `const` if you can.

Reply:
Fixed